src/AlarmClock.h

```diff
@@ -77,7 +76,7 @@ template<typename Duration> class AlarmClock {
       // the two atomics. Therefore stopwatch is needed to ensure the alarm
       // does not over sleep. 
       while (sw.ElapsedUs() < timeUsTillExpire) {
-         this_thread::sleep_for(microseconds(25));
+         std::this_thread::sleep_for(microseconds(25));
          if (mReset || mExit) {
             return false;
```

Comment:
@DylanSchneider
Please do a search for all atomic members and verify their usage

Example: (pseudocode)
```
if (mReset.load(aquire) || mExit.load(aquire)
```